eng/pipelines/git-tasks.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  - name: TaskType
+    displayName: Task Task
+    type: string
+    default: DeleteTag
+    values:
+      - DeleteTag
+
+  - name: Repository
+    displayName: Repository
+    type: string
+    values:
+      - Azure/azure-sdk-for-android
+      - Azure/azure-sdk-for-c
+      - Azure/azure-sdk-for-cpp
+      - Azure/azure-sdk-for-go
+      - Azure/azure-sdk-for-ios
+      - Azure/azure-sdk-for-java
+      - Azure/azure-sdk-for-js
+      - Azure/azure-sdk-for-net
+      - Azure/azure-sdk-for-python
+      - Azure/azure-sdk-tools
+
+  - name: TagName
+    displayName: "Tag Name"
+    type: string
+    default: ''
+
+jobs:
+- ${{if eq(parameters.TaskType, 'DeleteTag')}}:
+  - deployment: DeleteTag
+    displayName: "Delete ${{parameters.TagName}} tag from ${{parameters.Repository}} repository"
+    environment: git
```

Comment:
Is this a new environment? It seems pretty generic. We might want to consider a little better naming guidance for these so we know what they protect.

Reply:
I was considering having an environment per repository but that seemed like overkill. Got any suggestions on a name?

Reply:
Something like "github_operation_approvals" or something along those lines to give it some idea of what is implied without having to go dig through the environments.